client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -331,12 +366,3 @@ export const isLabelDataFetchError = ( state, orderId, siteId = getSelectedSiteI
 		arePackagesErrored( state, siteId )
 	);
 };
-
-export const getCountriesData = ( state, orderId, siteId = getSelectedSiteId( state ) ) => {
```

Comment:
Yeah, `eslint` fails if a function is used before it's defined. Before, `getCountriesData` wasn't used in this file, but now it is.

Reply:
Ah, right. Though it doesn't seem to be used anymore? Can't see another ref to `getCountriesData` in this file. But, it's all good to leave the placement as-is :)

Reply:
Argh, you're right. It's not used in this PR but it will be used in the next one, so this is a bit of a spoiler (and a side-effect of splitting the PRs after implementing everything 😂 )